doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -334,21 +332,21 @@ construct supports
 the dependent match annotations:
 
 .. prodn::
-   term += if @term is @pattern as @ident in @pattern return @term then @term else @term
+   term += if @term is @pattern as @ident in @pattern__ind return @term__ret then @term__then else @term__else
 
-As in ``let:``, the variable :token:`ident` (and those in the type pattern)
-are bound in the second :token:`term`; :token:`ident` is also bound in the
-third :token:`term` (but not in the fourth :token:`term`), while the
-variables in the first :token:`pattern` are bound only in the third
-:token:`term`.
+Similarly, this dependent ``if-is-then-else`` construct is just notation for
+:n:`match @term as @ident in @pattern__ind return @term__ret with @pattern as
+@ident => @term__then | _ => @term__else end`. In particular, :n:`@ident` is
```

Comment:
```suggestion
:n:`match @term as @ident in @pattern__ind return @term__ret with @pattern as @ident => @term__then | _ => term__else end`.
In particular, :n:`@ident` is
```
`:n:` retains internal line breaks, so better to put it all on one line.  (Would be even more readable if formatted on several lines with the patterns on separate lines, ie not run into the prose.)

<img width="1435" height="223" alt="image" src="https://github.com/user-attachments/assets/9428156b-83be-41af-80ed-3116b5b4d323" />